src/modules/ShortcutGuide/ShortcutGuide/shortcut_guide.h

```diff
@@ -64,7 +67,12 @@ class OverlayWindow
 
     struct DisabledApps
     {
-        PCWSTR name = L"disabled_apps";
-        std::wstring value = L"";
+        static inline PCWSTR name = L"disabled_apps";
+        std::wstring value;
     } disabledApps;
+
+    struct OpenShortcut
+    {
+        static inline PCWSTR name = L"open_shortcutguide";
+    } openShortcut;
```

Comment:
Should this have the `value` property as all other structs that identify a settings option?

Reply:
We use `OpenShortcut` only for sending telemetry so we don't need it